packages/nextjs-kit/src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as Header } from './header';
```

Comment:
Super minor, but why not make these named imports in header.tsx and footer.tsx themeselves?

Reply:
Open to changing it. I was following a multitude of guides at the same time and might have left some strange portions in there.